Modules/Loadable/Markups/qSlicerMarkupsModuleWidget.cxx

```diff
@@ -480,7 +480,7 @@ void qSlicerMarkupsModuleWidgetPrivate::setupUi(qSlicerWidget* widget)
   QTableWidgetItem *positionHeader = this->activeMarkupTableWidget->horizontalHeaderItem(this->columnIndex("Position"));
   positionHeader->setText("");
   positionHeader->setIcon(QIcon(":/Icons/Large/MarkupsPositionStatus.png"));
-  positionHeader->setToolTip(QString("Click in this column to set/unset control points."));
+  positionHeader->setToolTip(QString("Click in this column to define the position status of the control points."));
```

Comment:
"define the position status of the control points" sounds really complicated. I don't think many users could make sense of it. Do you have an idea how to simplify? Maybe something like "modify control point state"? Maybe we could add a bullet-point list in the tooltip (we do it at a few places), explaining each state:

"
- edit: ...
- undefined: ...
- missing: ...
- ...
"

Reply:
![image](https://user-images.githubusercontent.com/15837524/135538994-d829481a-b716-4f16-b9ee-7d98b9dcd342.png)